README.md

```diff
@@ -36,15 +36,29 @@ $ ssh-keygen -b 2048 -t rsa -f <a-path-to-save-the-file-to> -q -N ""
 
 ```.env
-IDENTIFIER=<an example identifier for your deployment> # eg. my-dev
-SSH_PRIVATE_KEY_LOCATION="<path/to/the/private/key>" # You created this in Step 3
-SSH_PUBLIC_KEY="<the contents of your public key>" # You created this in Step 3
-SSH_CONFIG_LOCATION="<path/to/your/ssh/config/file>" # This is optional, if not provided, a config file will be created for you
+SSH_PRIVATE_KEY_LOCATION="/home/me/ssh_key"
+SSH_PUBLIC_KEY="these are the contents of my public key"
+AWS_PROFILE="my-fave-aws-account"
 ```
 
+The variables recognised for this project are:
+
+| Variable                      	| Value(s)                                                                         	| Required 	| Default                                                             	| Description                                                                                                                                                                     	|
+|-------------------------------	|----------------------------------------------------------------------------------	|----------	|---------------------------------------------------------------------	|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------	|
+| `SSH_PRIVATE_KEY_LOCATION`    	| `<path/to/the/private/key>`                                                      	| ✅        	| N/A                                                                 	| You created this in Step 3                                                                                                                                                      	|
+| `SSH_PUBLIC_KEY`              	| `<The contents of your public key>`                                              	| ✅        	| N/A                                                                 	| You created this in Step 3                                                                                                                                                      	|
+| `AWS_PROFILE`                 	| `<Your named AWS CLI profile>`                                                   	| 🚫        	| default                                                             	| The AWS Profile to deploy to                                                                                                                                                    	|
+| `IDENTIFIER`                  	| `<An identifier for your deployment, e.g. 'my-dev'>`                             	| 🚫        	| dev                                                                 	| This is unique to your deployed stack                                                                                                                                           	|
```

Comment:
Why is Private Key required but Identifier not? Technically an ssh config doesn't need a private key specified (it just behaves better if you have lots of keys), But a unique stack name that one can look up seems pretty important.